shared/desktop/app/node.desktop.tsx

```diff
@@ -139,6 +149,18 @@ const createMainWindow = () => {
   SafeElectron.getIpcMain().on('remoteWindowWantsProps', (_, windowComponent, windowParam) => {
     mainWindow && mainWindow.window.webContents.send('remoteWindowWantsProps', windowComponent, windowParam)
   })
+
+  SafeElectron.getIpcMain().on('launchStartupURLIfPresent', () => {
```

Comment:
What do you think of making this something like `reduxLaunched` and stashing that in a module variable here? Then in `on('open-url'` you can look at that to decide whether to dispatch or stash the link.

Reply:
Thanks, does 02c044603116 look like what you're thinking?